src/Servers/IIS/test/IISIntegration.Tests/IISExtensionTests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
```

Comment:
@jkotalik are the files in this PR added to src/Servers/IIS/ supposed to be here?

Reply:
No, these seem duplicated. @ajaybhargavb can you remove these files and change the subdirectory name of https://github.com/aspnet/AspNetCore/tree/master/src/Servers/IIS/test/Microsoft.AspNetCore.Server.IISIntegration.Tests to IISIntegration.Tests?